Data/registers.json

```diff
@@ -663,6 +663,14 @@
           ],
           "width": 1,
           "width_64": 1
+        },
+        {
+          "is_encodable": false,
+          "registers": [
+            "IA32_KERNEL_GS_BASE"
```

Comment:
Hmm. This is not a traditional register, but just an MSR with a particular name. I think usually we don't model MSRs as registers, even if they may affect one instructions?

Reply:
Mhh yeah. In this case it's a rather important part of the instruction. I don't think modeling it as a register is a particularly good solution in this case, but everything else requires much more effort. Xed does the same and uses a pseudo register here.

Reply:
Hmm, yeah. Not pretty, but agree that it's probably the easiest approach here.